### PR TITLE
Removing system sitenotice

### PR DESCRIPTION
**Pull request by dmehus**
As per the previous sitenotice's length of time (approximately 24 hours) that @Reception123 and I discussed on Discord

### CODE REVIEW

**Comment by RhinosF1** [comment]
This is a lot more important so we discussed a week for this. I did say that I'd get figures on how many password changes were being made.

File: Sitenotice.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // and don't comment it out
 $wgMajorSiteNoticeID = 55;
 
-// if ( !$wmgSiteNoticeOptOut ) {
+/* // if ( !$wmgSiteNoticeOptOut ) {
```
Comment (Universal-Omega) [inline Sitenotice.php:13]: Can you please remove the `//` from here? While not really required I do think it simply looks better like that. Thanks!

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: Sitenotice.php (hunk 1 above)
Comment (dmehus) [reply Sitenotice.php:13]: @Universal-Omega Can you not make the change for me? For such a minor change, it seems a bit bureaucratic to have to make the requestor do it.

Comment (Universal-Omega) [reply Sitenotice.php:13]: sure, I can I just typically don't like modifying others' PRs, but I can do that if you want.

**Comment by dmehus** [comment]
@Universal-Omega Okay, thanks. :)

**Pull request merged**